<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQTClass</class>
 <widget class="QMainWindow" name="TestQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allBtn">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>22</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#allBtn{
background:white;
border:1px solid gray;
border-radius:5px;
padding:2px 4px;
color:gray;
}

QPushButton#allBtn:checked{
background:gray;
color:white;
}</string>
    </property>
    <property name="text">
     <string>all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>22</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#redBtn{
background:white;
border:1px solid rgb(255, 84, 92);
border-radius:5px;
padding:2px 4px;
color:rgb(255, 84, 92);
}

QPushButton#redBtn:checked{
background:rgb(255, 84, 92);
color:white;
}</string>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>22</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#blueBtn{
background:white;
border:1px solid #3288E8;
border-radius:5px;
padding:2px 4px;
color:#3288E8;
}

QPushButton#blueBtn:checked{
background:#3288E8;
color:white;
}</string>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>22</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#yellowBtn{
background:white;
border:1px solid rgb(255, 226, 6);
border-radius:5px;
padding:2px 4px;
color:rgb(255, 226, 6);
}

QPushButton#yellowBtn:checked{
background:rgb(255, 226, 6);
color:white;
}</string>
    </property>
    <property name="text">
     <string>yellow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setRedBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>22</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#setRedBtn{
border:1px solid rgb(255, 84, 92);
border-radius:5px;
padding:2px 4px;
background:rgb(255, 84, 92);
color:white;
}</string>
    </property>
    <property name="text">
     <string>set red</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>631</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
   background-color: transparent;
	border:1px solid #EFEFEF;
}</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>256</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeView {
   background-color: transparent;
	border:1px solid #EFEFEF;
}
QTreeView::item {
	height: 50px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestQT.qrc"/>
 </resources>
 <connections/>
</ui>
